### ISSUE TITLE
Reward Model Checkpoint selection?

### ISSUE THREAD

**Issue by backpropagate404** [rewind-reward/ReWiND#12]
Hi,

I reproduced your experiments, but I am a bit puzzled by the results I get for the training of the reward model. You set in the paper and in the codebase to run the reward model training 2k steps on metaworld.
But here are my wandb curves, all the metrics decrease past 1.5k steps. At 2k steps I get lower performances than the ones reported in the paper. I use the config set by default in your codebase, and seed=42.

<img width="885" height="1060" alt="Image" src="https://github.com/user-attachments/assets/cf1604e1-350a-404b-8d9b-8a0c2266ab0c" />

<img width="1380" height="349" alt="Image" src="https://github.com/user-attachments/assets/dd30039a-f916-4a91-b7ef-b8bfa6206081" />

As you don't have any validation set, how do you do the checkpoint selection? To get similar performances as the ones reported in the paper, I should select here the checkpoint 1.5k, but as I use the same config as you, with the same seed, I wonder if you also selected an early checkpoint?
Is it expected that the metrics go down like this from 1.5k steps?

Thanks a lot.
